Mobile Test Runner for Jama/Mobile Test Runner for Jama/LoginViewController.swift

```diff
@@ -117,6 +118,7 @@ extension LoginViewController: EndpointDelegate{
             //Notify user of unauthorized reload the view on the main thread
             DispatchQueue.main.async {
                 self.reloadViewWithUsernameAndInstanceSaved()
+                self.viewDidLoad()
```

Comment:
I assume the `self.userNameTextBox.delegate = self` lines are needed from the viewDidLoad function to make the keyboard work properly. 

We decided that since the fields are still empty and the cursor remains when receiving that error, it makes sense for the keyboard to still be active since the user should be typing things to retry. What do you think?